CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [4.3.0](https://github.com/sonata-project/SonataFormatterBundle/compare/4.2.0...4.3.0) - 2020-09-24
+### Added
+- [[#477](https://github.com/sonata-project/SonataFormatterBundle/pull/477)] Added support for symfony/options-resolver:^5.1 ([@phansys](https://github.com/phansys))
+- [[#477](https://github.com/sonata-project/SonataFormatterBundle/pull/477)] Added support for symfony/property-access:^5.1 ([@phansys](https://github.com/phansys))
+
+### Changed
+- [[#472](https://github.com/sonata-project/SonataFormatterBundle/pull/472)] Allow FOSCKEditor dependency to ^2.0 ([@bmaziere](https://github.com/bmaziere))
```

Comment:
```suggestion
- [[#477](https://github.com/sonata-project/SonataFormatterBundle/pull/477)] Support for symfony/options-resolver:^5.1 ([@phansys](https://github.com/phansys))
- [[#477](https://github.com/sonata-project/SonataFormatterBundle/pull/477)] Support for symfony/property-access:^5.1 ([@phansys](https://github.com/phansys))
- [[#472](https://github.com/sonata-project/SonataFormatterBundle/pull/472)] Support for friendsofsymfony/ckeditor-bundle:^2.0 ([@bmaziere](https://github.com/bmaziere))
```

Reply:
IMO we dont need dupicate `Add` and should keep the same standard for added dependencies.

Reply:
In that case, could you please open a PR in the [dev-kit](https://github.com/sonata-project/dev-kit/blob/2b85361c91245c6a3e426c343726a14d898eeaad/templates/project/.github/PULL_REQUEST_TEMPLATE.md.twig#L40)?